lib/index.d.ts

```diff
@@ -0,0 +1,600 @@
+import { OrderItem } from "./parsers/orderItems";
+
+type InitParams = {
+    accessKeyId: string;
+    authToken?: string;
+    host?: string;
+    merchantId: string;
```

Comment:
It's best practice to define required fields before optional fields.

Reply:
i just sorted the entire thing alphabetically, but sorting by optional/required then alphabetically is probably a better idea.

Reply:
updated latest commit